CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ set (PYTHON_VERSION "2.7" CACHE STRING "Target version of python to find")
 option (PYLIB_INCLUDE_SONAME "If ON, soname/soversion will be set for Python module library" OFF)
 option (PYLIB_LIB_PREFIX "If ON, prefix the Python module with 'lib'" OFF)
 option (USE_FIELD3D "Use Field3D if found" ON)
+option (USE_OPENVDB "Use OpenVDB if found" ON)
 option (USE_FFMPEG "Use FFmpeg if found" ON)
```

Comment:
Should OPENVDB_ROOT be directly exposed as a cached variable here, so that it's more apparent for people running 'ccmake', for example, to interactively see and edit all the build-time options?